extensions/amp-iframe/amp-iframe.md

```diff
@@ -84,3 +84,27 @@ Here are some factors that affect how fast the resize will be executed:
 - Whether the resize is triggered by the user action;
 - Whether the resize is requested for a currently active IFrame;
 - Whether the resize is requested for an IFrame below the viewport or above the viewport.
+
+#### Iframe Click-To-Play
```

Comment:
Make a note above where it says 600px to indicate this. E.g. "position must be 600px or it should be a click-to-play iframe as described below"